dags/tti_aggregate.py

```diff
@@ -0,0 +1,72 @@
+import sys
+import os
+
+from datetime import datetime, timedelta
+from airflow.providers.postgres.operators.postgres import PostgresOperator
+from airflow.models import Variable
+from airflow.decorators import dag, task
+from airflow.macros import ds_add, ds_format
+
+import logging
+import pendulum
+try:
+    repo_path = os.path.abspath(os.path.dirname(os.path.dirname(os.path.realpath(__file__))))
+    sys.path.insert(0, repo_path)
+    from dags.dag_functions import task_fail_slack_alert
+except:
+    raise ImportError("Cannot import slack alert functions")
+    
+LOGGER = logging.getLogger(__name__)
+logging.basicConfig(level=logging.DEBUG)
+
+doc_md = """
+
+### The Daily TTI Aggregation DAG
+
+This DAG runs daily to aggregate citywide TTI and downtown TTI after the pull_here dag finishes.
+Slack notifications is raised when the airflow process fails.
+
+"""
+DAG_NAME = 'tti_aggregate'
+DAG_OWNERS = Variable.get('dag_owners', deserialize_json=True).get(DAG_NAME, ["Unknown"]) 
+
+# Slack alert
+SLACK_CONN_ID = 'slack_data_pipeline'
```

Comment:
I discovered this variable assignment is defunct! See: https://github.com/CityofToronto/bdit_data-sources/blob/245c53152a68ef5cc0e57a0956bcfd98b2b408a4/dags/dag_functions.py#L82-L85

Reply:
:meow_wow: